EIPS/eip-1620.md

```diff
@@ -1,296 +1,185 @@
 ---
 eip: 1620
 title: ERC-1620 Money Streaming
-author: Paul Berg (@PaulRBerg)
+author: Paul Razvan Berg (@PaulRBerg)
 discussions-to: https://github.com/ethereum/EIPs/issues/1620
-status: Draft
+status: Last Call
 type: Standards Track
 category: ERC
 created: 2018-11-24
+requires: 20
+review-period-end: 2020-09-24
 ---
 
 ## Simple Summary
-Money streaming represents the idea of continuous payments over a finite period of time. Block numbers are used as a proxy of time to continuously update balances.
+
+We define money streaming as continuous payments over a finite period of time. Block timestamps are used as an on-chain proxy for time.
```

Comment:
> Block timestamps are used as an on-chain proxy for time.

This seems like an implementation detail, which is not appropriate for a standardized interface.  If we remove that, we are left with:
> We define money streaming as continuous payments over a finite period of time

Which seems like the definition of a new phrase "money streaming" and an EIP isn't the right place to try to define new terminology.

Consider rewriting this to more clearly define what this is trying to standardize.  If you are just trying to standardize a phrase, then I recommend closing this and pursuing that elsewhere.

Reply:
Well, they have code, so it's not just terminology. Should be something like, "Standard for sending continuous payments over time."